broadcasting/06_RTSP_Introduction.md

```diff
@@ -0,0 +1,22 @@
+# Introduction
```

Comment:
We will need to add information that we will efectively change the program written in the previous chapters so that to support RTSP stream instead of RTMP

Reply:
I think that this
```
In the next couple chapters, we would like to show you a modification of the current RTMP to HLS converter, that will allow us to convert an RTSP stream instead.
```
indicates, that we will change the architecture of the solution.